protocol.c

```diff
@@ -42,10 +42,10 @@ static uint8_t next_report=REQUEST_STATUS_REPORT;
 // Directs and executes one line of formatted input from protocol_process. While mostly
 // incoming streaming g-code blocks, this also directs and executes Grbl internal commands,
 // such as settings, initiating the homing cycle, and toggling switch states.
-static void protocol_execute_line(char *line)
+static uint8_t protocol_execute_line(char *line)
 {
   protocol_execute_runtime(); // Runtime command check point.
-  if (sys.abort) { return; } // Bail to calling function upon system abort
+  if (sys.abort) { return STATUS_ABORT; } // Bail to calling function upon system abort
```

Comment:
Put the return on the next line. 
I try to format all the code that we touch correctly.